stable/traefik/values.yaml

```diff
@@ -2,6 +2,8 @@
 image: traefik
 imageTag: v1.2.1
 serviceType: LoadBalancer
+# Static IP Address pass --set loadBalancerIP=false from command line if running mulitiple 
```

Comment:
`loadBalancerIP=false` doesn't seem to make sense. Either set it or leave it empty. Also, there's a typo in `mulitiple`. I'd just remove the comment.